components/CarouselWithImages.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+const props = defineProps({
+  images: {
+    type: Array,
+    required: true
+  }
+})
+</script>
+
+<template>
+  <v-carousel v-if="props?.images && props.images?.length" 
+    width="100%"
+    height="445"
+    :show-arrows="false"
+    hide-delimiter-background
+    cycle
+    class="custom-carousel"
+  >
+    <template v-for=" (prop, index) in props.images" :key="index">
+      <v-carousel-item v-if="prop.src?.length && prop.alt?.length" 
+        :src="prop.src"
+        :alt="prop.alt"
+        cover
+        height="421"
+      >
+      </v-carousel-item>
+    </template>
+  </v-carousel>
+</template>
+
+<style>
+.custom-carousel .v-carousel__controls {
+  max-height: 20px;
+  background-color: var(--white-color);
+}
+
+.custom-carousel .v-carousel__controls .v-btn {
+  width: var(--carousel-dot-width);
+  height: var(--carousel-dot-width);
+  color: var(--black-color);
+}
+
+.custom-carousel .v-carousel__controls .v-btn .v-icon {
+  font-size: 1rem;
+  opacity: 1;
+}
+
+.custom-carousel .v-btn--active .v-btn__overlay {
+  background-color: transparent;
+}
+
+.custom-carousel .v-btn:not([class$='--active']) .v-icon {
```

Comment:
Suggested change:

.custom-carousel .v-btn:not(.v-btn--active) .v-icon 

Here I am not sure, but perhaps this change will be more specific and clear? It will only exclude elements that have the class v-btn--active, which is more precise and prevents accidental matches of other classes ending in --active.

Reply:
@PiotrDrechsler Yes, that's make sense, I'll change it.